.github/workflows/main.yml

```diff
@@ -105,7 +105,7 @@ jobs:
     - name: Autotools Configure 
       if: matrix.generator == 'autogen'
       run: |
-        sh ./autogen.sh
+        sh ./bin/chkmanifest
         mkdir "${{ runner.workspace }}/build"
```

Comment:
I would probably run both autogen and chkmanifest explicitly. Running autogen inside of chkmanifest is odd and an artifact. I'd like to remove it since autogen won't run on systems where the autotools aren't installed, but that will take some fixin'.

Reply:
Creating an issue for this, as it may affect other processes already using chkmanifest. This change is simply to prevent a instance where the MANIFEST is missing a file, which causes problems with building from a source tarball.

Reply:
Pull request was ammended to run both explicitly for now.  I'll merge it.